src/bucketizers/timebasedBucketizer.ts

```diff
@@ -15,6 +14,8 @@ export default class TimebasedBucketizer implements Bucketizer {
     private readonly k: number = 4;
     private readonly minBucketSpan: number = 300000;
 
+    private readonly logger = getLogger("TimebasedBucketizer");
```

Comment:
That's the wrong logger. Would have been better if you would have started from #40